castervoice/lib/ccr/core/nav.py

```diff
@@ -298,7 +298,9 @@ class Navigation(MergeRule):
         Choice("semi", {
             "dock": ";",
             "doc": ";",
-            "sink": ""
+            "sink": "",
+            "(com | comma)": ",",
+            "(deck | deckle | cole)": ":",
```

Comment:
Can you pull in and invert the punctuation dictionary here? I should be able to change the punctuation dictionary and have it work here. to invert a dict do something like: 

```
inv_tpb = {v: k for k, v in text_punc_dict.iteritems()}
```

Reply:
Why would I need to invert the dictionary? Isn't it already in the proper orientation for this purpose?

Reply:
You're adding single syllable versions here, which is fine because they're prefixed with "tell" so ambiguity is unlikely. However, the elements "deckle", "comma" etc are in the punc dictionary and should be referenced from there. To access "deckle", we'd need to do `inv_tdb[":"]` so the new command would be `"(deck | cole | " + inv_tdb[":"] + ")"`

Reply:
I now see what you did in your latest update. Yes that approach works nicely.